Assets/_BForBoss/_Character/Scripts/PlayerMovementBehaviour.cs

```diff
@@ -69,6 +73,17 @@ public void SetControlConfiguration()
             characterLook.controllerVerticalSensitivity = _inputConfiguration.ControllerVerticalSensitivity;
         }
         
+        public void ModifyPlayerSpeed(float modificationMultiplier)
+        {
+            _unModifiedPlayerSpeed = maxWalkSpeed;
+            maxWalkSpeed += maxWalkSpeed * modificationMultiplier;
```

Comment:
Does `maxWalkSpeed *= modificationMultiplier`

Reply:
This could work with `SlowDown` but with `SpeedUp` then the multiplier of 0 makes the player motionless, and the multiplier of 1 will make the player move at its normal speed.

So I think by doing `+=` keeps it more intuitive